client/rest/src/plugins/rosetta/CatapultProxy.js

```diff
@@ -114,6 +114,9 @@ export default class CatapultProxy {
 				this.fetch('blocks/1')
 			]);
 
+			if (this.cache)
+				return this.readCacheProperty(propertyName);
```

Comment:
Is there still a race between the assignment of the cache object and ``currencyMosaicId`` getting formatted correctly? 🤔

Reply:
should not be - after the await, the code should execute on main thread and there shouldn't be parallel executions afaikt